tests/test_browser.py

```diff
@@ -2582,3 +2582,93 @@ def test_modularize(self):
         ''' % code)
         self.run_browser('a.html', '...', '/report_result?0')
 
+  # Test that the emscripten_ atomics api functions work.
```

Comment:
Made the test harness report skip on the pthread tests if the browser doesn't support them. Also, if a user runs a pthread-compiled page in a browser that doesn't support pthreads, the execution will abort immediately at startup.